.github/workflows/pr-lint.yml

```diff
@@ -0,0 +1,19 @@
+name: PR Lint
+on: pull_request
+
+jobs:
+  markdownlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: install dependencies
+        run: |
+          npm install --prefix tools/linters
+      - name: run linter
+        run: |
+          ./tools/linters/node_modules/.bin/markdownlint -c ./tools/linters/markdownlint.json ./*.md text/**/*.md
```

Comment:
use `npx markdownlint` instead of explicitly reaching out to `node_modules/.bin`

Reply:
Done!